srcpkgs/vscode/patches/disable-crash-reporter.patch

```diff
@@ -14,7 +14,7 @@ So, disable it entirely. No cookies for you, Visual Studio Code team.
  3 files changed, 1 insertion(+), 20 deletions(-)
 
 diff --git a/src/bootstrap-fork.js b/src/bootstrap-fork.js
-index f7b07e49255..576c82f23e8 100644
+index c11168a2bc7..5978959dc2a 100644
 --- a/src/bootstrap-fork.js
 +++ b/src/bootstrap-fork.js
 @@ -37,9 +37,6 @@ if (process.env['VSCODE_PARENT_PID']) {
```

Comment:
This patch was just rebased, could you remove the changes if the old version still applies?

Reply:
done.